homeassistant/components/watson_tts/tts.py

```diff
@@ -0,0 +1,137 @@
+"""Support for IBM Watson TTS integration."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.tts import PLATFORM_SCHEMA, Provider
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_URL = 'watson_url'
+CONF_APIKEY = 'watson_apikey'
+ATTR_CREDENTIALS = 'credentials'
```

Comment:
You could use that from `const.py` by core and that helps user because all configs look the same.